README.md

```diff
@@ -1,7 +1,7 @@
 [![Ansible-lint status](https://github.com/aalaesar/install_nextcloud/actions/workflows/ansible-lint.yml/badge.svg)](https://github.com/aalaesar/install_nextcloud/actions?workflow=Ansible%20Lint)
 [![YAML-lint status](https://github.com/aalaesar/install_nextcloud/actions/workflows/yamllint.yml/badge.svg)](https://github.com/aalaesar/install_nextcloud/actions?workflow=Yaml%20Lint)
 
-# install_nextcloud
+# Ansible role: Nextcloud
```

Comment:
Since keeping the old name for now, can you also revert that or set the old name as 
```suggestion
# Ansible role: install_nextcloud
```
?

Thank you :)